opentelemetry-api/tests/test_loader.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from importlib import reload
+import sys
+import unittest
+import os
+
+from opentelemetry import loader
+from opentelemetry import trace
+
+DUMMY_TRACER = None
+
+class DummyTracer(trace.Tracer):
+    pass
+
+def get_opentelemetry_implementation(type_):
+    global DUMMY_TRACER #pylint:disable=global-statement
+    assert type_ is trace.Tracer
+    DUMMY_TRACER = DummyTracer()
+    return DUMMY_TRACER
+
+#pylint:disable=redefined-outer-name,protected-access,unidiomatic-typecheck
+
+class TestLoader(unittest.TestCase):
+
+    def setUp(self):
+        reload(loader)
+        reload(trace)
+
+        # Need to reload self, otherwise DummyTracer will have the wrong base
+        # class after reloading `trace`.
+        reload(sys.modules[__name__])
+
+
```

Comment:
It looks like `pylint` doesn't care about blank lines... do you think it'd be overkill to run `flake8` with lint too?

Reply:
For your consideration: https://github.com/open-telemetry/opentelemetry-python/pull/46. 😄